content/en/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you.md

```diff
@@ -10,11 +10,11 @@ weight: 20
 
 The `dockershim` component of Kubernetes allows to use Docker as a Kubernetes's
 {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}.
-Kubernetes' built-in `dockershim` component was deprecated in release v1.20.
+Kubernetes' built-in `dockershim` component was removed in release v1.24.
```

Comment:
Is the intention to publish this after the release? If yes, then the past tense makes sense. If not, we might have to reword it.

Reply:
We will publish this change (which targets _dev-1.24_) as part of the v1.24 release.